multiview/example/example_gcca.py

```diff
@@ -0,0 +1,27 @@
+import sys
```

Comment:
Don't have a good spot of putting this but can you not delete example.py and the test? think it serves as a good template for now. Also lets us know that docs are building since it's the only function being autodoced at the moment.

Reply:
yeah left them in but they were failing pystyle checks which is why I removed them in the first place. Have since fixed said pystyle problems.

Reply:
Ooh i see my bad and thanks for fixing